ooniprobe/View/TestResults/Footer/UploadFooterViewController.h

```diff
@@ -0,0 +1,15 @@
+#import <UIKit/UIKit.h>
+#import "Result.h"
+#import "Measurement.h"
+
+@interface UploadFooterViewController : UIViewController
```

Comment:
done

Reply:
Please document also the properties explaining their meaning

Reply:
It's sufficient to have a single doc sentence before each line explaining the purpose

Reply:
I'm gonna document only upload_all as the others result, measurement, titleLabel are self explanatory, what do you think?

Reply:
I think a short piece of doc for every public attribute is better. Tip: if you use `///` rather than `//` for documentation comments, Xcode recognised that and provides more help.